### DISCUSSION TITLE
Persistent Data Labels

### DISCUSSION THREAD

**Discussion by mpw-eng** [grafana/grafana#38170] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Please add the ability to place persistent data labels on panels, similar to those shown on the Excel chart image attached. 

**Why is this needed**:
Grafana is an excellent tool for analysing historical process data.  Having the ability to place notes, with data values and timestamps would allow Grafana panels to be used in technical reports, showing what happened and when, with key events highlighted with data labels.

Currently (Version 7.3.0), allows annotations, but it's not possible to show the text behind multiple annotations at once.  To document key chart features, several screenshots of the same chart with the tooltip/cursor at different timestamps is necessary.

![image](https://user-images.githubusercontent.com/53257685/97657075-eb9f2900-1aa3-11eb-90b9-4eeee00fe34d.png)
![image](https://user-images.githubusercontent.com/53257685/97657856-e6db7480-1aa5-11eb-827b-8ec7ab6fc894.png)

**Comment by hugohaggmark** [comment]
Thank you for adding this request @mpw-eng, is this something we should consider for the new Graph NG #27848 @dprokop?

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
